core/whats-new/whats-new-in-21/vb/Program.vb

```diff
@@ -0,0 +1,7 @@
+Imports System
+
+Module Program
+    Sub Main(args As String())
+        Console.WriteLine("Hello World!")
+    End Sub
+End Module
```

Comment:
Do I understand it correctly that [VB doesn't support `Span<T>`](https://sharplab.io/#v2:DYLgbgRgNALuCmAnAlgMwJ4B8CSBbADgPaIwDOABAMrqkzy4CwAUAAoCuEwyAxuQLKEAJm2DxyLRIQDmiAIaMm5JeI5delDv1nIAdgAoAlM2UnyAEWS5yOtrghIKAXnIA5eAHdy2HXSlI9AKwADAbkAN4AvsamStExFlb4siTkzm6elEn6APKoXj7wfogGejZ2DlDkAIxBlTVGisoAojqCVBzMLW0CwqJAA=)? Though showing Hello World here doesn't seem like the right solution to me.

Reply:
The VB code won't show in the doc, @svick, except for the Brotli example. The VB code doesn't compile because the compiler sees Span(Of T) as a nested obsolete class.

Reply:
Then is there a reason to have the Program.vb here at all?

Reply:
Actually, no. I was leaving it as a placeholder for when Span<T> is supported in VB. But it's just as well to remove it.